pandas/io/excel/_base.py

```diff
@@ -40,7 +40,7 @@
 
 Parameters
 ----------
-io : str, ExcelFile, xlrd.Book, path object or file-like object
+io : str, ExcelFile, xlrd.Book, path object, binary data or file-like object
```

Comment:
```suggestion
io : str, ExcelFile, xlrd.Book, path object, binary data, or file-like object
```

Reply:
BTW, I believe we still need an update to `io.rst`.

Reply:
what do you think should be added? an example of reading binary files?

Reply:
On second look, I actually don't think we need anything here in `io.rst`.  Thanks for asking!

Also, looking at the documentation change here, now that I think about it, what do you think about just doing this:

```suggestion
io : str, bytes, ExcelFile, xlrd.Book, path object, or file-like object
```

Reply:
`bytes` looks much better but please let's keep the final comma out :P

Reply:
> but please let's keep the final comma out :P

In this repository, we put commas in those situations.

Reply:
As you prefer and sorry if it sounded bad, didn't mean to! I had looked at other read_* docstrings so they might need updating as well.